components/server/src/installation-admin/telemetry-data-provider.ts

```diff
@@ -21,7 +21,6 @@ export class InstallationAdminTelemetryDataProvider {
         try {
             const data: TelemetryData = {
                 installationAdmin: await this.installationAdminDb.getData(),
-                totalUsers: await this.userDb.getUserCount(true),
```

Comment:
Heads up! I consider the installation admin be not used, thus removing the `getUserCount` usage here seems to be save.